examples/refsi/hal_refsi/external/refsidrv/source/refsidrv/refsi_device_g.cpp

```diff
@@ -28,9 +28,7 @@
 // does not have dedicated (TCIM) memory for storing kernel exeutables, a memory
 // window is set up to map this memory area to a reserved area in DMA.
 constexpr const uint64_t REFSI_ELF_BASE = 0x10000ull;
-// The upper region can be up to the tdcm start point at 0x10000000
-// We will make it approx 128MB to give some latitude
-constexpr const uint64_t REFSI_ELF_SIZE = (1 << 27) - REFSI_ELF_BASE;
+constexpr const uint64_t REFSI_ELF_SIZE = (1 << 20) - REFSI_ELF_BASE;
```

Comment:
Does reducing this risk us rejecting large binaries that were previously accepted?

Reply:
In theory, yes. This cuts the binary size down from 128MiB to 1MiB. However, I've not seen binaries anywhere near that big (most are about 1KiB). The only reason the change from `1 << 20` to `1 << 27` was made in the first place was, I think, to work around this issue, which is why I reverted it.

I'm honestly not sure if we should keep it. 1MiB does seem a bit low for complicated kernels.

Reply:
Ah, https://github.com/codeplaysoftware/oneapi-construction-kit/pull/92, thanks, that makes sense. I do worry that it might be a bit low but if it's what we had before, I'm okay with it, we can always increase it again later if needed.